app/features/onboarding/actionTypes.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * The type of (redux) action that hides active onboarding.
+ *
+ * {
+ *     type: HIDE_ACTIVE_ONBOARDING,
+ *     parentComponent: string
```

Comment:
How about calling this "section" instead?

Reply:
Done